# target servo
geometry_msgs/Transform target

---
# If the service request was acknowledged
bool success
